x-pack/plugins/security_solution/cypress/objects/rule.ts

```diff
@@ -11,6 +11,8 @@ export const totalNumberOfPrebuiltRules = rawRules.length;
 
 export const totalNumberOfPrebuiltRulesInEsArchive = 127;
 
+export const totalNumberOfPrebuiltRulesInEsArchiveCustomRule = 145;
```

Comment:
Maybe I'm dense, but the link between this name and the archive isn't super clear, here. Not a blocker, but perhaps we should qualify it with `...InCustomRuleWithTimelineArchive` or something like that.

Reply:
Similarly, `totalNumberofPrebuildRulesInEsArchive` is also unclear now for a similar reason: which archives does that number pertain to?